tensorflow_privacy/privacy/membership_inference_attack/codelabs/codelab_privacy_risk_score.ipynb

```diff
@@ -0,0 +1,816 @@
+      "with 0.6 as the threshold on privacy risk score, the precision-recall pair is (0.6118458884416331, 0.2128)\n",
+      "with 0.5 as the threshold on privacy risk score, the precision-recall pair is (0.5705128205128205, 0.712)\n",
+      "\n",
+      "For slice type: CLASS=7\n",
+      "with 0.6 as the threshold on privacy risk score, the precision-recall pair is (0.6164383561643836, 0.18)\n",
+      "with 0.5 as the threshold on privacy risk score, the precision-recall pair is (0.5659122874312748, 0.8852)\n",
+      "\n",
+      "For slice type: CLASS=8\n",
+      "with 0.5 as the threshold on privacy risk score, the precision-recall pair is (0.5507837390085383, 0.8644)\n",
+      "\n",
+      "For slice type: CLASS=9\n",
+      "with 0.6 as the threshold on privacy risk score, the precision-recall pair is (0.6332288401253918, 0.202)\n",
+      "with 0.5 as the threshold on privacy risk score, the precision-recall pair is (0.5841666666666666, 0.701)\n",
+      "\n",
+      "For slice type: CORRECTLY_CLASSIFIED=True\n",
+      "with 0.5 as the threshold on privacy risk score, the precision-recall pair is (0.5220848110408277, 0.6059067835717582)\n",
+      "\n",
+      "For slice type: CORRECTLY_CLASSIFIED=False\n",
+      "with 0.7 as the threshold on privacy risk score, the precision-recall pair is (0.7104532829471387, 0.3436936936936937)\n",
+      "with 0.6 as the threshold on privacy risk score, the precision-recall pair is (0.6816679555870855, 0.624024024024024)\n",
+      "with 0.5 as the threshold on privacy risk score, the precision-recall pair is (0.6396320935483625, 0.8274774774774775)\n",
+      "\n"
+     ]
+    }
+   ],
+   "source": [
+    "from tensorflow_privacy.privacy.membership_inference_attack.dataset_slicing import get_single_slice_specs\n",
+    "from tensorflow_privacy.privacy.membership_inference_attack.dataset_slicing import get_slice\n",
+    "slicing_spec = SlicingSpec(\n",
+    "                            entire_dataset = True,\n",
+    "                            by_class = True,\n",
+    "                            by_classification_correctness = True\n",
+    "                            )\n",
+    "input_slice_specs = get_single_slice_specs(slicing_spec, 10)\n",
+    "for single_slice_spec in input_slice_specs:\n",
+    "    \n",
+    "    attack_input_slice = get_slice(input, single_slice_spec)\n",
+    "    risk_score_results = mia._compute_privacy_risk_score(attack_input_slice)\n",
+    "    print(f\"For slice type: {str(risk_score_results.slice_spec)}\")\n",
+    "    risk_score_results.print_results()\n",
+    "    print()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": []
+  }
+ ],
+ "metadata": {
+  "colab": {
+   "collapsed_sections": [],
+   "last_runtime": {
+    "build_target": "//learning/deepmind/public/tools/ml_python:ml_notebook",
+    "kind": "private"
+   },
+   "name": "Membership inference codelab",
```

Comment:
Should we rename this to make it clear that this is about the privacy risk score?

Also, can we rename the filename to privacy_risk_score_codelab to make it consistent with other files?

Reply:
Fixed these issues!